nuget-diff.cake

```diff
@@ -61,6 +61,8 @@ if (!diffs.Any()) {
 		var newPath = temp.CombineWithFilePath(newName);
 
 		CopyFile(diff, newPath);
+		// for github PR summary (markdown files are not accepted, so copy to text files)
+		CopyFile(diff, $"{diff}.txt");
```

Comment:
This doesn't specify a directory, does it end up in the same spot as the .md files?

Reply:
it is the same file, but with txt extension for upload:
```
tree ./output/api-diff/Xamarin.Google.MLKit.Vision.Common/monoandroid90 
./output/api-diff/Xamarin.Google.MLKit.Vision.Common/monoandroid90
├── Xamarin.Google.MLKit.Vision.Common.dll.diff.md
└── Xamarin.Google.MLKit.Vision.Common.dll.diff.md.txt
```